client.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/lightninglabs/loop/loopdb"
 	"github.com/lightninglabs/loop/swap"
 	"github.com/lightninglabs/loop/sweep"
+	"github.com/lightningnetwork/lnd/routing/route"
```

Comment:
commit message: non-blocking, but generally when we touch multiple packages, can just prefix commit name with `multi:`, saves some time :)

Reply:
I usually cut it at 2 fwiw, but no hard rule